_config.yml

```diff
@@ -0,0 +1,18 @@
+title: OpenEAC Alliance
+markdown: kramdown
+defaults:
+  -
+    scope:
+      path: ""
+    values:
+      layout: "page"
+plugins:
+  - jekyll-redirect-from
+include: ['_pages']
+defaults:
+  - scope:
```

Comment:
This list-item can be put into the `defaults` block above after the unscoped one.